smt/examples/airfoil_parameters/learning_airfoils_parameters.py

```diff
@@ -0,0 +1,211 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from smt.surrogate_models.genn import GENN, load_smt_data
+import csv
+from sklearn.model_selection import train_test_split
+
+"""
+Predicting Airfoil Aerodynamics through data by Raul Carreira Rufato and Prof. Joseph Morlier
+This is a tutorial to determine the aerodynamic coefficients of a given airfoil using GENN in SMT (Other models could be used as well) 
```

Comment:
Other -> other

Reply:
"""
Predicting Airfoil Aerodynamics through data by Raul Carreira Rufato and Prof. Joseph Morlier
This is a tutorial to determine the aerodynamic coefficients of a given airfoil using GENN in SMT (other models could be used as well)